README.md

```diff
@@ -99,6 +99,26 @@ if err != nil {
     fmt.Println("Token is not valid:", token)
 }
 ```
+## Support key cacher interface
+### Persistent key cacher:
+
+``` func NewJWKClient(options JWKClientOptions, extractor RequestTokenExtractor) *JWKClient```
```

Comment:
Might be more aligned with the rest of README to actually put full examples here.